### DISCUSSION TITLE
Filter/Sort Repositories

### DISCUSSION THREAD

**Discussion by AlexHedley** [community/community#18886] [Mobile]
Could there be an option added to filter repos by Organisation?
Add a search
Add a sort

Other pages have it, not sure why this one doesn't.

**Comment by Iron-Ham** [comment]
Hi @AlexHedley, thank you for the feedback! 
I've added this to an internal tracking issue for the team.

**Reply by AlexHedley** [reply]
I've been trying this out on the beta.

Is there a reason it wasn't added to all Repo screens?

**Reply by Iron-Ham** [reply]
The mobile app uses the GraphQL API to perform its network operations. Not every `repositories` field within the API is sortable or filterable by the same metrics; some are scoped to their specific context.

**Reply by AlexHedley** [reply]
Home (tab) | My Work | Repositories
- Type

Home (tab) | My Work | Organisations | [Org] | Repositories
- Search
- Type
- Language
- Sort

Profile (tab) | Repositories
- Search
- Type
- Language
- Sort

**Reply by Iron-Ham** [reply]
Ah, I see what you mean now – apologies.
As far as I can tell, it was a conscious decision to limit filtering and search from within `Home`, but it's something I can bring up to the team in more detail Monday.

**Reply by AlexHedley** [reply]
Thanks :-)
